train.py

```diff
@@ -118,8 +122,11 @@ def main(args=None):
             try:
                 optimizer.zero_grad()
 
-                classification_loss, regression_loss = retinanet([data['img'].cuda().float(), data['annot']])
-
+                if torch.cuda.is_available():
+                    classification_loss, regression_loss = model([data['img'].cuda().float(), data['annot']])
```

Comment:
are you sure this is right? why the change from `retinanet` to `model` ?

Reply:
It was a typo and I have changed it.

Reply:
Thanks!